swap_meet/electronics.py

```diff
@@ -1,2 +1,13 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+    def __init__(self, id=None, type="Unknown", condition=0):
+        super().__init__(id, condition)
+        self.type = type
+
+
+
```

Comment:
While space can be useful for readability, it's good to try and keep just one empty line between functions!